pkgs/by-name/np/nps/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  nix,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nps";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "OleMussmann";
+    repo = "nps";
+    tag = "v${version}";
+    hash = "sha256-q/PkigsNAI7MCmeDFBMGuZJFXVL95pQCNOVhNvBH9dc=";
+  };
+
+  cargoHash = "sha256-o+H7Dv8BH2cBqSiXCIYUveAL9KS5A0cbBI3E2Q9Thag=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "sha256-MThyvhzZXRM4l0K8csLDldMVKiDxKZ5EIFATGVpGpVc=";
```
We are moving to our new cargo fetcher nixpkgs-wide.

---

Thanks for the heads-up. Using `useFetchCargoVendor` and a regenerated `cargoHash` now.